src/SelectField/SelectField.jsx

```diff
@@ -20,9 +20,15 @@ function getStyles(props) {
 }
 
 const SelectField = React.createClass({
+  displayName: 'SelectField',
 
   propTypes: {
     /**
+     * opens DropDownMenu and triggers focus of TextField
+     */
+    autoFocus: React.propTypes.bool,
```

Comment:
`React.PropTypes.bool?` - what else?

`openImmediately?` - to keep it simple i didn't intend to rename things.
and as it's called like that in DropDownMenu i just passed it.